src/cljs/athens/types/dispatcher.cljs

```diff
@@ -1,8 +1,9 @@
-(ns athens.views.blocks.types.dispatcher)
+(ns athens.types.dispatcher)
 
 
 (defmulti block-type->protocol
   "Takes `:block/type` value and converts it to `BlockTypeProtocol` to be used for rendering"
   (fn [block-type _args-map]
+    #_(println "block-type->protocol:" (pr-str block-type))
```

Comment:
This looks a good place to introduce the feature flag for tasks. Return `:default` instead of the block-type.

Reply:
Tasks for now are created by adding the block type manually, so there's no other trigger like slash commands etc. So there's no other place to add the feature flag check.